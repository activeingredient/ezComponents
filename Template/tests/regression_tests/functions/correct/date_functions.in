{use $timestamp}
1. {date_format_timestamp( "Y-m-d", 1147362767 )}
2. {date_format_timestamp( "Y-m-d H:m:i", 1147362767 )}
3. {if (date_current_timestamp() == $timestamp)}correct
{/if}
