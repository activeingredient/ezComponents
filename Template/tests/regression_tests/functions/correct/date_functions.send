<?php
$v = new ezcTemplateVariableCollection();
$v->timestamp = time();

return $v;
?>
