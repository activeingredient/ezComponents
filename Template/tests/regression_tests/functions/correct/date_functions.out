1. 2006-05-11
2. 2006-05-11 15:05:52
3.  correct     correct 